src/System.Management.Automation/help/HelpCommands.cs

```diff
@@ -202,6 +219,20 @@ public SwitchParameter Online
 
         private bool _showOnlineHelp;
 
+        /// <summary>
+        /// This parameter,if true, will direct get-help cmdlet to
+        /// display the help content using Microsoft.PowerShell.Pager on alternate screen buffer.
+        /// </summary>
+        [Parameter]
+        public SwitchParameter Paged
+        {
+            get;
+            set;
+        }
+
+        private Pager pager;
```

Comment:
Our convention for private fields:
```suggestion
        private Pager _pager;
```
Also we could add property to exclude null checks and initializations from code below:
`private Pager Pager => _pager ??= new Pager();`

Reply:
Fixed with a property as suggested.